src/webextension/list/manage/components/upgrade-account.js

```diff
@@ -0,0 +1,65 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+ import { Localized } from "fluent-react";
+import PropTypes from "prop-types";
+import React from "react";
+import { connect } from "react-redux";
+
+import Button from "../../../widgets/button";
+
+import styles from "./upgrade-account.css";
+
+export function UpgradeAccount({mode, ...props}) {
+  if (mode !== "guest") {
+    return null;
+  }
+
+  const doCreate = async () => {
+    browser.runtime.sendMessage({
+      type: "upgrade",
+      action: "signup",
+    });
+  };
+  const doSignIn = async () => {
+    browser.runtime.sendMessage({
+      type: "upgrade",
+      action: "signin",
+    });
+  };
+
+  return (
+    <section className={styles.upgrade}>
+      <Localized id="homepage-upgrade-title">
+        <h2>uPGRADe</h2>
+      </Localized>
+      <Localized id="homepage-upgrade-description">
+        <p>uPGRADe yOUr lOCKBOx</p>
+      </Localized>
+      <menu>
+        <Localized id="homepage-upgrade-action-create">
+          <Button id="homepage-upgrade-action-create"
```

Comment:
Avoid using IDs if you can help it. While I'm sure we could create sufficiently-unique IDs for single-use React components, I'd rather just not go there at all. If this is for tests and you need some help getting tests working without having IDs on elements, let me know and I can point you at some examples.

Reply:
This was for testing, so pointers would be much appreciated.

Reply:
This is how I normally do it: https://github.com/mozilla-lockbox/lockbox-extension/blob/master/test/list/manage/components/item-details-test.js#L47-L48

Enzyme lets you find the `<Localized/>` element by its ID, and then you can just find the `<Button>` it wraps from there.